bundles/org.openhab.binding.mycroft/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mycroft"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="mycroft">
+		<label>Mycroft</label>
+		<description>A Mycroft instance</description>
+
+		<channels>
+			<channel id="listen" typeId="listen-channel"/>
+			<channel id="speak" typeId="speak-channel"/>
+			<channel id="utterance" typeId="utterance-channel"/>
+			<channel id="player" typeId="player-channel"/>
+			<!-- <channel id="volume" typeId="volume-channel"/> -->
+			<channel id="volume_mute" typeId="volume-mute-channel"/>
+			<channel id="volume_duck" typeId="volume-duck-channel"/>
+			<channel id="full_message" typeId="full-message-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Mycroft Bus Host Address</label>
```

Comment:
See https://www.openhab.org/docs/developer/bindings/config-xml.html#supported-contexts

Please add a context `network-address` here.

Reply:
Done, I didn't know this, I will try to remember. thx